src/orquestra/sdk/_base/_config.py

```diff
@@ -578,37 +463,10 @@ def read_config_names() -> List[str]:
     """
     try:
         with filelock.FileLock(_get_config_directory() / LOCK_FILE_NAME, timeout=3):
-            return _read_config_names()
-    except filelock.Timeout:
-        raise IOError(
-            "Could not acquire the lock for the config file at "
-            f"'{get_config_file_path()}' "
-            "- does another function or process currently hold it? "
-            "If you're calling `read_config_names` from a context that has already "
-            "acquired the lock, you may want to look into using `_read_config_names` "
-            "instead."
-        )
-
-
-def _read_config_names() -> List[str]:
-    """
-    Reads the names of all configurations stored in the configuration file.
-
-    This function is intended for internal use in cases where we already have a
-    filelock for the config file. If you aren't wrapping the call to this function in
-    `with filelock...`, you should probably be using `read_config_names` instead.
-
-    Returns:
-        list: a list of strings, each containing the name of a saved configuration. If
-            the file does not exist, returns an empty list.
-    """
-
-    try:
-        config_file = _open_config_file()
+            config_file = _open_config_file()
+            return [name for name in config_file.configs]
```

Comment:
**question**: what's the reason for removing the human-friendly error message on lock timeout?

Reply:
TLDR is that we use those locks all around the config code, and this is the only place it had this error, which was fine but inconsistent. I dont think it matters that much honestly anyway.